src/content_delegate.js

```diff
@@ -361,9 +361,20 @@ ContentDelegate.prototype.onDocumentViewSubmit = function (event) {
       // dcd (and presumably others) trigger this code path.
       var reader = new FileReader();
       reader.onload = function() {
+          let html=reader.result;
+          // check if we have an HTML page which redirects the user to the PDF
+          // this was first display by the Northern District of Georgia
+          // https://github.com/freelawproject/recap/issues/277
+          const redirectResult = Array.from(html.matchAll(/(?:window\.location\s=\s")(.*)(?:\")/g));
+          if(redirectResult && redirectResult.length>0) {
+            let url = redirectResult[0][1];
+            html = '<style>body { margin: 0; } iframe { border: none; }' +
+                 '</style><iframe src="' + url +
+                 '" width="100%" height="100%"></iframe>';
+          }
```

Comment:
Maybe it's just me, but it's weird to close the style tag and open the iframe tag on the same line, when the whole thing is multiline. Perhaps a template literal? I 'm not up on my CSS enough to know if all of these explicit specifications are necessary? Also can't we `<iframe/>`? Maybe not.

On the other hand, I guess this is what the existing code does above, so I suppose I should shut up about it.

The `let`/`var` police are probably going to show up and complain about treating `html` as if it were a block-scoped variable and reusing it from the prior block, because the variable declaration doesn't match the control flow, although it works fine because it is hoisted. But perhaps those cops should be appeased.

Reply:
Yeah, exactly. I was just matching the style on line 355 vs. putting my own style in. (I'd prefer to use template literals too.)  Bigger picture, I didn't spot a style guide or any linting. Let me know if you have any interest in those, and I'm happy to make a separate PR/contribution of those.

Reply:
On the scoping of `HTML`, I converted the declaration on line 355 to be a `let` making them both scoped to the block.

Reply:
Feel free to swap the prior one to a template literal in a separate commit and rewrite history to have it come first. Or could be a separate PR if you want.

Style is...complicated but there is not a clear and rigorous guide here, so.